chat_box/main.py

```
@@ -0,0 +1,121 @@
+def filter_output(disease, allergy, model_reply):
+    forbidden_words = []
+    if disease in FORBIDDEN_BY_DISEASE:
+        forbidden_words += FORBIDDEN_BY_DISEASE[disease]
+    if allergy in FORBIDDEN_BY_ALLERGY:
+        forbidden_words += FORBIDDEN_BY_ALLERGY[allergy]
+
+    for item in forbidden_words:
+        if item.lower() in model_reply.lower():
+            return 0
+        
+    return 1;
```

Ruff (0.13.1)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)